numpy/core/_add_newdocs.py

```diff
@@ -3939,17 +3939,22 @@
 
     Constructs Python bytes showing a copy of the raw contents of
     data memory. The bytes object can be produced in either 'C' or 'Fortran',
```

Comment:
Note that `'Fortran'` probably shouldn't be in quotes as that implies that it is a valid input for the `order` parameter.

Reply:
Correcting it to 'F'